src/Aether/Aether.fs

```diff
@@ -2,7 +2,7 @@
 
 open System
 
-/// Optics
+//  Optics
```

Comment:
I changed it to simple comment, to remove it from the documentation comment of `Lens`.
Same for `Isomorphism` below.